functions/Get-DbaDbOrphanUser.ps1

```diff
@@ -102,7 +102,8 @@ function Get-DbaDbOrphanUser {
                                     SqlInstance  = $server.DomainInstanceName
                                     DatabaseName = $db.Name
                                     User         = $user.Name
-                                }
+                                    SMOUser      = $user
```

Comment:
```suggestion
                                    SmoUser      = $user
```

Reply:
@potatoqualitee What do you like? The *RoleMember commands use "SMOUser" so I used it here. "SmoUser" is ok for me as well - but maybe then change it everywhere?

Reply:
Yes, today I like SmoUser too 😅

Reply:
Should I change the name in *RoleMember commands also? Within this PR?

Reply:
You can if you'd like, yes. Once tests pass, I'll merge them all.

Reply:
Ok, done.